nixos/doc/manual/release-notes/rl-1703.xml

```diff
@@ -17,6 +17,13 @@ has the following highlights: </para>
   </listitem>
 
   <listitem>
+    <para>Setting capabilities on programs is now supported with a
+    <literal>setcap-wrapper</literal> functionality. This
```

Comment:
This is a bit confusing because there is no `setcap-wrapper` anymore. Probably better to say "the setuid wrapper now also supports setting capabilities".